app/models/course.rb

```diff
@@ -0,0 +1,13 @@
+# Class describing a course
+class Course < ApplicationRecord
+  attribute :is_hidden, :boolean, default: true
```

Comment:
This isn't necessary. This just duplicates the column definition in the courses table and will just cause problems if we ever update one without changing the other.